packages/grafana-ui/src/components/ColorPicker/SeriesColorPickerPopover.tsx

```diff
@@ -1,17 +1,33 @@
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useState, useEffect } from 'react';
 
 import { ColorPickerPopover, ColorPickerProps } from './ColorPickerPopover';
 import { PopoverContentProps } from '../Tooltip/Tooltip';
 import { Switch } from '../Switch/Switch';
 import { withTheme } from '../../themes/ThemeContext';
+import appEvents from '../../../../../public/app/core/app_events';
 
 export interface SeriesColorPickerPopoverProps extends ColorPickerProps, PopoverContentProps {
   yaxis?: number;
   onToggleAxis?: () => void;
 }
 
 export const SeriesColorPickerPopover: FunctionComponent<SeriesColorPickerPopoverProps> = props => {
-  const { yaxis, onToggleAxis, color, ...colorPickerProps } = props;
+  const [color, setColor] = useState(props.color);
```

Comment:
Not sure I understand the state ownership here. Seems like right now there are multiple ways how things can change. From props, from events, there is `props. onColorChange` which is not actually used here but we check if it exists so I assume it also changes the color from some where. Could we just lift the state upstream? Who ever is owning the `color` state should probably be the one handling any color changes.